.travis.yml

```diff
@@ -1,62 +1,58 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
```

Comment:
Indeed. I reverted all code-style related changes to the `lib/Raven` directory and I reverted PHPCS rules to PSR-2 only